docs/_posts/2021-12-08-tongzhanggui-customer-platform.md

```diff
@@ -0,0 +1,209 @@
+死不掉也活不好，谁能改变中国100万快消二批的命运？
+
+播报文章
+
+![pic](/assets/2021/12-08-tongzhanggui-customer-platform/46d4b0230deffe6a686efef8daa60089.gif)
+
+作者：陈思廷
+
+![pic](/assets/2021/12-08-tongzhanggui-customer-platform/aab533d22346508474a1b7c956cd35c4.png)
+
+**二批，快消厂家无不欲扁平之而后快**
```

Comment:
Use header 2 for titles (##)

Reply:
fix it

> Thanks for posting this great news! Please follow my reviews, and rename `tong-liqun` to `liqun-tong` to follow our naming convention.

liqun-tong has exists。 i delete the md